components/public-api-server/pkg/server/integration_test.go

```diff
@@ -13,151 +13,101 @@ import (
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(err).String())
+	prebuildClient := v1connect.NewPrebuildsServiceClient(http.DefaultClient, srv.HTTPAddress(), connect.WithInterceptors(auth.NewClientInterceptor("some-token")))
 
-	_, err = client.CreateAndStartWorkspace(ctx, connect.NewRequest(&v1.CreateAndStartWorkspaceRequest{}))
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(err).String())
+	_, err = prebuildClient.GetPrebuild(ctx, connect.NewRequest(&v1.GetPrebuildRequest{}))
+	requireErrorStatusCode(t, connect.CodeUnimplemented, err)
 
-	_, err = client.StartWorkspace(ctx, connect.NewRequest(&v1.StartWorkspaceRequest{}))
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(err).String())
+	_, err = prebuildClient.GetRunningPrebuild(ctx, connect.NewRequest(&v1.GetRunningPrebuildRequest{}))
+	requireErrorStatusCode(t, connect.CodeUnimplemented, err)
 
-	_, err = client.GetActiveWorkspaceInstance(ctx, connect.NewRequest(&v1.GetActiveWorkspaceInstanceRequest{}))
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(err).String())
-
-	_, err = client.GetWorkspaceInstanceOwnerToken(ctx, connect.NewRequest(&v1.GetWorkspaceInstanceOwnerTokenRequest{}))
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(err).String())
-
-	stream, err := client.ListenToWorkspaceInstance(ctx, connect.NewRequest(&v1.ListenToWorkspaceInstanceRequest{}))
+	listenToStatusStream, err := prebuildClient.ListenToPrebuildStatus(ctx, connect.NewRequest(&v1.ListenToPrebuildStatusRequest{}))
 	require.NoError(t, err)
-	stream.Receive()
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(stream.Err()).String())
+	_ = listenToStatusStream.Receive()
+	err = listenToStatusStream.Err()
+	requireErrorStatusCode(t, connect.CodeUnimplemented, err)
 
-	logsStream, err := client.ListenToImageBuildLogs(ctx, connect.NewRequest(&v1.ListenToImageBuildLogsRequest{}))
+	listenToLogsStream, err := prebuildClient.ListenToPrebuildLogs(ctx, connect.NewRequest(&v1.ListenToPrebuildLogsRequest{}))
 	require.NoError(t, err)
-	logsStream.Receive()
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(logsStream.Err()).String())
-
-	stopStream, err := client.StopWorkspace(ctx, connect.NewRequest(&v1.StopWorkspaceRequest{}))
-	require.NoError(t, err)
-	stopStream.Receive()
-	require.Equal(t, connect.CodeUnimplemented.String(), connect.CodeOf(stopStream.Err()).String())
+	_ = listenToLogsStream.Receive()
+	err = listenToLogsStream.Err()
+	requireErrorStatusCode(t, connect.CodeUnimplemented, err)
 }
 
-func requireErrorStatusCode(t *testing.T, expected codes.Code, err error) {
-	require.Error(t, err)
-	st, ok := status.FromError(err)
-	require.True(t, ok)
-	require.Equalf(t, expected, st.Code(), "expected: %s but got: %s", expected.String(), st.String())
+func requireErrorStatusCode(t *testing.T, expected connect.Code, err error) {
+	t.Helper()
+	if expected == 0 && err == nil {
```

Comment:
Connect doesn't define the default Error Code value as OK, so we have to do this extra check..